django_project/config/settings/base.py

```diff
@@ -89,9 +82,8 @@
 
 USE_TZ = True
 
-
-# Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/1.8/howto/static-files/
-
 STATIC_URL = '/static/'
-STATIC_ROOT = '/server/%(project_name)s'
+STATIC_ROOT = os.path.join(PROJECT_ROOT, STATIC_URL.strip("/"))
+
+MEDIA_URL = STATIC_URL + "media/"
+MEDIA_ROOT = os.path.join(PROJECT_ROOT, *MEDIA_URL.strip("/").split("/"))
```

Comment:
https://docs.python.org/3/library/stdtypes.html#str.strip

Essentially, strip only removes `/` one at a time from either end until it hits a character that isn't `/`. That means the `/` between "static" and "media" will be spared.